jsapp/js/projects/customViewRoute.tsx

```diff
@@ -100,18 +102,32 @@ function CustomViewRoute() {
           onClick={exportAllData}
         />
 
+        {selectedAssets.length === 0 && (
+          <div className={styles.actions}>
+            <ProjectQuickActionsEmpty />
+          </div>
+        )}
+
         {selectedAssets.length === 1 && (
-          <div className={styles.quickActions}>
+          <div className={styles.actions}>
             <ProjectQuickActions asset={selectedAssets[0]} />
```

Comment:
I think it's more normal to retain the selection after a bulk action. e.g. Gmail bulk "mark read", Finder bulk rename or tag – both retain the selection.

More:

- For us it's a little moot right now since we only support delete. But if we supported >1 archive it might be nice to retain the selection so you can see where they all went.
- I definitely want to retain the selection if the list is 'updated' due to ~~pagination~~ infinite scroll

This selection UI in general feels a little incomplete, as we don't support ranged selection or deselect-all (clear selection button) yet. If we add bulk archive / unarchive it would be nice to have an undo. And it might be nice to have a "X projects selected" indicator somewhere.

Reply:
We could add the indicator here:
<img width="524" alt="Screenshot 2023-08-25 at 22 58 08" src="https://github.com/kobotoolbox/kpi/assets/2521888/8dd1d31c-7d53-460f-a7e2-18f674ffd70b">

Reply:
I feel it would be nice to have it (not saying we should add it right now), as after I archived a project, it went outside the visible area. By mistake user could think that there is nothing selected, and could select another project and got confused or did some action unknowingly.